releases-beta/how-to-guides/create-custom-workflow-parameters.md

```diff
@@ -0,0 +1,63 @@
+# Create Custom Workflow Parameters
+
+Custom workflow parameters allow you to define and manage key-value pairs that can be accessed during your software build and deployment process. This ensures that your release pipeline can use specific configurations for different environments, such as staging, production, or custom setups.
+
+## Generate Workflow Parameters in Your Project or Environment Config
```

Comment:
Let's break it down into 2 sections to make it easier for reader to find what they need:
- Setting workflow parameters for your Build workflow: This section provides the instruction to change the settings in "Project Config". The existing content LGTM.
- Setting workflow parameters for your Deploy workflow: This section provides the instruction to change the settings in "Environment Config". The content can be simpler because the rest is pretty much the same